src/components/header/header.js

```diff
@@ -0,0 +1,44 @@
+import { useRef } from "react";
+
+import PropTypes from "prop-types";
+
+import * as styledHeader from "./header.module.scss";
+
+const Header = props => {
+  const headerRef = useRef(null);
+  const {
+    sticky = true,
+    stickyMobile = false,
+    maxWidth,
+    backgroundColor,
+    boxShadow,
+    justify,
+    border,
```

Comment:
if you don't use these props, please remove them

Reply:
Done